src/cls/rgw/cls_rgw_types.cc

```diff
@@ -39,7 +41,7 @@ void cls_rgw_obj_key::decode_json(JSONObj *obj) {
 void rgw_bucket_dir_entry_meta::generate_test_instances(list<rgw_bucket_dir_entry_meta*>& o)
 {
   rgw_bucket_dir_entry_meta *m = new rgw_bucket_dir_entry_meta;
-  m->category = 1;
+  m->category = RGWObjCategory::Main;
```

Comment:
cool;  although what does "Main" mean?

Reply:
Added documentation in src/cls/rgw/cls_rgw_types.h.